src/package.json

```diff
@@ -36,12 +42,20 @@
     "test:e2e": "./node_modules/.bin/jest --testMatch '**/e2e/?(*.)test.ts' --passWithNoTests",
     "test:infra": "./node_modules/.bin/jest --testMatch '**/infra/?(*.)test.ts' ",
     "api": "JEST_JUNIT_OUTPUT_NAME=api-report.xml ./node_modules/.bin/jest --runInBand --testPathPattern=tests/api/",
-    "test:api": "npm run compile && npm run api"
+    "test:api": "npm run compile && npm run api",
+    "kill:dynamodblocal": "cd tests/contract; ./scripts/kill-dynamodb-local.sh",
+    "start:dynamodblocal": "cd tests/contract; ./scripts/setup-dynamodb-local.sh",
+    "start:contract": "npm-run-all -s start:dynamodblocal wait-start:contract",
+    "wait-start:contract": "wait-on tcp:127.0.0.1:8000 && cd tests/contract && ts-node ./app.ts",
+    "test:provider": "./node_modules/.bin/jest --testMatch '**/contract/F2fCriProvider.test.ts' --coverage=false",
+    "test:contract": "wait-on tcp:127.0.0.1:3000; cd tests/contract; npm run test:provider; npm run kill:dynamodblocal",
```

Comment:
I am calling the kill:dynamodblocal script after the test execution. Is this okay or will impact the pipeline execution once its integrated there?